specification/monitor/resource-manager/microsoft.insights/preview/2018-06-01-preview/examples/createOrUpdateGuestDiagnosticSettings.json

```diff
@@ -0,0 +1,241 @@
+{
+    "parameters": {
+        "subscriptionId": "187f412d-1758-44d9-b052-169e2564721d",
+        "resourceGroupName": "Default-ResourceGroup",
+        "diagnosticSettingsName": "SampleDiagSetting",
+        "api-version": "2018-06-01-preview",
+        "diagnosticSettings": {
+            "location": "Global",
+            "tags": {},
+            "properties": {
+                "osType":  "windows",
```

Comment:
Can we capitalize the first letter of the OSes -- Windows/Linux?

Reply:
Done